tests/integration/conftest.py

```diff
@@ -42,8 +43,10 @@ def pytest_addoption(parser):
         '--tls', default=False, action='store_true',
         help='Use tls')
     parser.addoption(
-        "--runslow", action="store_true", default=False, help="run slow tests"
-    )
+        "--runslow", action="store_true", default=False, help="run slow tests")
+    parser.addoption(
+        '--elle-cli', default='../elle-cli/target/elle-cli-0.1.4-standalone.jar',
```

Comment:
I think it'd be nice to run `elle` tests only when this option is given. Similar to above `--runslow` option and `--tls` option. Because, when we want to run tests on a new machine, we'll need to install those new packages. It'll be painful. Let's make `elle` test optional.

Reply:
done